tests/unit/conftest.py

```diff
@@ -128,18 +128,19 @@ def invalid_json_config(tmpdir_factory,
 
 
 @pytest.fixture(scope='session')
-def simple_json_config(tmpdir_factory,
-                       default_global_properties,
-                       default_testcase_properties,
-                       default_testcase_elements):
+def simple_json_config(tmpdir_factory):
     """config sample"""
     config = \
 """
 {
-    "test_cycle": "pike",
-    "project_id": 12345,
-    "module_hierarchy": ["one","two","three"],
-    "path_to_test_exec_dir": "{{ '' }}"
+    "zigzag": {
+        "test_cycle": "pike",
+        "project_id": "12345",
+        "build_url": "https://bar.com/foo",
+        "build_number": "78",
```

Comment:
I'm confused. How is zigzag going to know about "build_number" or "build_url"? That information is captured is captured by "pytest-zigzag". However, in this recently merged [PR](https://github.com/rcbops/pytest-zigzag/pull/17/files#diff-d5a2a31eae2614c742550345ccf43d8eL13) the requirement has been removed.

Reply:
This isn't the best example of how the two sections of the config file are used.  [The molecule example config](https://github.com/rcbops/zigzag/blob/f46331c1966710e8a2617902e3776eb8f542f0a0/zigzag/data/configs/molecule-config-example.json) is a better example of how it would be used in practice.  I think there are two key concepts that I think are important to understand this.  

1. ANY environment variable can be captured within the `pytest_zigzag_env_vars` section.
2. Those previously captured environment variables are then referenced in `zigzag` section.  [On this line](https://github.com/rcbops/zigzag/blob/f46331c1966710e8a2617902e3776eb8f542f0a0/zigzag/data/configs/molecule-config-example.json#L33) we reference the variable `RE_JOB_BRANCH` and use it to define the config `project_branch`.  The possible configs are currently documented [in the configuring zigzag section of the README](https://github.com/rcbops/zigzag/blob/f46331c1966710e8a2617902e3776eb8f542f0a0/README.rst)

Reply:
The configs that are noted as required are validated by the [json schema file](https://github.com/rcbops/zigzag/blob/f46331c1966710e8a2617902e3776eb8f542f0a0/zigzag/data/schema/zigzag-config.schema.json#L27)